gallery_dl/extractor/boosty.py

```diff
@@ -219,6 +219,107 @@ def items(self):
             yield Message.Queue, url, user
 
 
+class BoostyDirectMessagesExtractor(BoostyExtractor):
+    """Extractor for boosty.to direct messages"""
+    subcategory = "direct-messages"
+    directory_fmt = ("{category}", "{user[blogUrl]} ({user[id]})",
+                     "direct-messages")
+    pattern = BASE_PATTERN + r"/app/messages\?dialogId=(\d+)"
+    example = "https://boosty.to/app/messages?dialogId=123456"
+
+    def items(self):
+        """Yield direct messages from a given dialog ID."""
+        dialog_id = self.groups[0]
+        response = self.api.dialog(dialog_id)
+        sign = response.get("signedQuery", "")
+        messages = response.get("messages", {}).get("data", [])
```

Comment:
Please avoid this `obj.get("key", {})` pattern.

Wrap it in a try-except block, since it's going to succeed most of the time anyway, or at least use `obj.get("key") or {}` to avoid needlessly constructing a new dict.